generate-dm.sh

```diff
@@ -3,7 +3,7 @@
 cd dm/proto
 
 echo "generate dm protobuf code..."
-GOGO_ROOT=${GOPATH}/src/github.com/gogo/protobuf
+GOGO_ROOT=/Users/imtbkcat/go/src/github.com/gogo/protobuf
```

Comment:
😕

Reply:
I use `gvm` to manage local go version, so `GOPATH` can not accessed by this shell script, I will fix this right now. 😂